openapi/platform/platform.yaml

```diff
@@ -1423,6 +1423,46 @@ paths:
                         type: string
                         format: uri
                         description: The address of the newly created business location
+              examples:
+                Return Example (with ID):
+                  value:
+                    id: AG-1234567
+                    type: salesAccounts
+                    attributes:
+                      tags:
+                        - tag1
+                        - tag2
+                      customerIdentifier: User-defined-id-123123
+                      name: Company Example
+                      address:
+                        line1: 109 8th Street E.
+                        line2: Suite 23
```

Comment:
If you're going to use a real address in docs, I'd use a famous landmark like the CN Tower. `290 Bremner Blvd, Toronto, ON M5V 3L9`